<launch>
    <!-- Usage: roslaunch walker_brain open_fridge.launch -->
    <!-- Before launching this, launch prepare_push_cart.launch -->

    <param name="/use_sim_time" value="true" />

    <!-- Could be 6, 7, 8 -->
    <arg name="task_id" default="7"/>

    <node pkg="walker_brain" type="walk_helper.py" name="walk_helper" output="screen">
    </node>

    <!-- Here we use the same helper as open fridge but with different parameters -->
    <node pkg="walker_brain" type="open_fridge_helper.py" name="open_fridge_helper" output="screen">
        <!-- In webots world frame -->
        <!-- cart vertical surface center: x:-0.89, z:3.5; Robot: x:-1.33495, z:3.5 -->
        <param name="x_offset" value="0.445" />
        <param name="y_offset" value="0" />
    </node>

    <node pkg="walker_brain" type="open_fridge_node" name="open_fridge_node" output="screen">
        <param name="tree_file" value="$(find walker_brain)/tree/push_cart_$(arg task_id).xml" />
    </node>

</launch>